.github/workflows/CI.yml

```
@@ -58,10 +58,10 @@ jobs:
         uses: actions/checkout@v3
 
       - name: Install Cocoapods
-        run: gem install cocoapods --no-document --quiet
+        run: ruby install cocoapods --no-document --quiet
         
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)